framework/base/src/types/interaction/tx.rs

```diff
@@ -1,6 +1,7 @@
 use crate::{
     api::CallTypeApi,
     contract_base::BlockchainWrapper,
+    proxy_imports::{EgldOrEsdtTokenIdentifier, EgldOrEsdtTokenPaymentRefs, ManagedTypeApi},
```

Comment:
`proxy_imports` is just for proxies, we shouldn't use it in the framework
it's annoying that the auto-completer defaults to it
we'll purge them from the framework, but it's ok, doesn't need to be in thie PR